core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthCheckServiceBuilder.java

```diff
@@ -250,13 +259,23 @@ public HealthCheckServiceBuilder updatable(HealthCheckUpdateHandler updateHandle
         return this;
     }
 
+    /**
+     * Sets whether the specified {@link ActionType} is enabled or not for the
+     * {@link #build() HealthCheckService}. All {@link ActionType}s are disabled by default.
+     */
+    public HealthCheckServiceBuilder transientServiceAction(ActionType actionType, boolean enable) {
```

Comment:
Can use varags by removing `enable` from parameters?
```java
enableTransientServiceAction(ActionType... actionTypes);
```

Reply:
Had a chat with @ikhoon and we decided to leave it as it is because the default value might be changed in the future.

Reply:
How about this:

- If `transientServiceActions()` is not invoked at all, use the default settings.
- If invoked at least once, use the specified ones.